app/helpers/works_helper.rb

```diff
@@ -96,12 +96,16 @@ def marked_for_later?(work)
     reading && reading.toread?
   end
 
+  def markasread_link(work)
+    link_to ts("Mark as Read"), marktoread_work_path(work)
+  end
+
   def marktoread_link(work)
     link_to ts("Mark for Later"), marktoread_work_path(work)
   end
 
-  def markasread_link(work)
-    link_to ts("Mark as Read"), marktoread_work_path(work)
+  def removemarktoread_link(work)
```

Comment:
Right now, there are three buttons:

- Mark as Read, which incorrectly uses the old path (line 99-101)
- Mark for Later, which correctly uses the old path (line 103-105)
- Remove mark for later, which uses the new path (line 107-109)

Why aren't we changing the existing Mark as Read to use the new path instead of adding a new one with different phrasing? Mark as Read is the same thing as removing from Marked for Later.